CDP4WebServices.API/Services/ChangeLog/ChangeLogService.cs

```diff
@@ -306,7 +306,7 @@ public bool TryAppendModelChangeLogData(NpgsqlTransaction transaction, string pa
                             operationData.Update.Add(modelLogEntryClasslessDTO);
                         }
 
-                        this.OperationProcessor.Process(operationData, transaction, partition);
+                        this.OperationProcessor.Process(operationData, transaction, partition, false);
```

Comment:
Not sure why appending ModelChangeLog is exempt from reading data from cache. Should we take a look at that together? Could do a lot, because there can be reads there.

Reply:
agreed, in the mean time i have noticed that not all integration tests pass, have more work to do to get this to work 100%. Suffice to say, we have to be conscious when we query the CACHE and when we query the VIEWs.